.github/workflows/linux_builds.yml

```diff
@@ -39,6 +39,7 @@ jobs:
             ${{github.job}}-${{env.GODOT_BASE_BRANCH}}-${{github.ref}}-${{github.sha}}
             ${{github.job}}-${{env.GODOT_BASE_BRANCH}}-${{github.ref}}
             ${{github.job}}-${{env.GODOT_BASE_BRANCH}}
+        continue-on-error: true # if the cache load fails continue build (very common issue)
```

Comment:
Is there a reason we don't do this for all jobs?

Reply:
cache is the only optional job

Reply:
What do you mean? The jobs are things like "linux-editor" and "linux-editor-sanitizers". There are multiple jobs per YML file.

Reply:
Ah indeed, there's a few other jobs which need to same flag on their own cache restore steps.

Reply:
ah okay, yes I get what you mean, I missed the caches further down in the yaml.

Reply:
fixed

Reply:
There's one more for Linux, it has three jobs.